### ISSUE TITLE
Make use of dynamic HUDs to re-implement buildable status displays.

### ISSUE THREAD

**Issue by Viech** [Unvanquished/Unvanquished#524]
The current buildable status displays have no text on them and have their style/formatting hardcoded, which makes them hard to understand and even harder to maintain.

**Comment by bmorel** [comment]
What do you mean by "current buildable status displays"?

**Comment by Viech** [comment]
> What do you mean by "**current** buildable status displays"?

He asks, seven years later. :grin: 

Buildable status displays are those shown when you are a granger or hold a ckit. These are to my knowledge rendered in 3D and the layout is done by the client side gamelogic.

**Comment by bmorel** [comment]
> He asks, seven years later. ￼

Yep, I occasionally dig into old issues to see if stuff can be closed, and I was not a contributor 7 years ago :)

If you mean the health-bar, with the (possible) un-powered and deconstruction-candidate marks, then I guess this link (https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/blob/9a7bc181205bc2c4470f531dd7c233213dc800bf/ui/assets/alien/buildstat.cfg) confirms it's done by rendering engine, possibly in 3D (I do not know if engine mixes a 2D glProgram and a 3D glProgram, so I won't take the easy shortcut to say it's in 3D).

If I understand correctly, the request issue is to have this implemented with libRocket?

**Comment by Viech** [comment]
> If I understand correctly, the request issue is to have this implemented with libRocket?

Yes, precisely. This would allow to scale and style them and add text when needed.
